apps/storybook/src/stories/nft/NFTCard.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { NFTCard as NFTCard$ } from "@geist/ui-react/components/nft/nft-card";
+import type { Meta, StoryObj } from "@storybook/react";
+import { withWagmiProvider } from "#stories/decorators/wagmi.tsx";
+
+interface ERC721CardStoriesProps {
+	tokenId: string;
+	contractAddress: string;
+}
+
+function ERC721CardStories({
```

Comment:
can we create 2 stories for 2 separate smart vs dumb components 
(similar to `TransactionTable` vs `TransactionTableWithDetails`

end goal is we don't dictate the data source